packages/shared/src/models/IPlaygroundMessage.ts

```diff
@@ -18,11 +18,18 @@
 
 import type { ModelOptions } from './IModelOptions';
 
-export interface ChatMessage {
+export interface Message {
   id: string;
+  timestamp: number;
+}
+
+export interface ErrorMessage extends Message {
+  message: string;
```

Comment:
IMO this is not explanatory enough, might have something like `error: string` instead ?

Reply:
updated 👍